app/views/task/task_pdf.pdf.erb

```diff
@@ -112,8 +112,20 @@ No Tutor % Supervisor's Name
   <%
     if file[:type] == 'code'
       if (File.extname(file[:path])[1..-1]) != 'ipynb'
+      # for code files (except Jupyter Notebook), always use files with the .tmp suffix
+      # if the file has not been modified, it will be a symlink instead
+      # Jupyter Notebooks are not processed with line wraps
+      dir = File.dirname(file[:path])
+      name = File.basename(file[:path])
+      pygments_lang = Task.pygments_lang(File.extname(file[:path])[1..-1])
+      wrapped_file = File.join(dir, "#{name}.tmp")
   %>
-\inputminted[breaklines,linenos,breakanywhere,tabsize=4]{<%= Task.pygments_lang(File.extname(file[:path])[1..-1]) %>}{<%= file[:path] %>}
+    <% unless File.symlink?(wrapped_file) %>
```

Comment:
If we re-wrote the original we would have to think of a different way to handle this...